### ISSUE TITLE
Whole row clikable as a link

### ISSUE THREAD

**Issue by agustibr** [CodiTramuntana/ct_table_for#3]
Add an option to make the whole row of the column clikable

An example:

```
<tbody>
    <tr class='clickable-row' data-href='url://'>
        <td>Blah Blah</td> <td>1234567</td> <td>£158,000</td>
    </tr>
</tbody>


jQuery(document).ready(function($) {
    $(".clickable-row").click(function() {
        window.location = $(this).data("href");
    });
});
```

**Comment by ananichev** [comment]
@agustibr Hi! I did it in the #4. Please review it. Thanks

**Comment by tramuntanal** [comment]
Hi @ananichev we'll review it this Wednesday. Thanks for your collaboration!

**Comment by tramuntanal** [comment]
Hi @ananichev we'll review it this Wednesday. Thanks for your collaboration!

**Comment by agustibr** [comment]
Closed issue in commit 641348e37c989ce4262d04ead23382e7bb944788
